### PR TITLE
Add NuxtJS recipe

### PR DESCRIPTION
**Pull request by xlanex6**
Simple recipe HOW TO install and config notif with nuxtJS

### CODE REVIEW

File: recipes/README.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Follow these recipes to easily use Notyf in React, Angular or Vue.
 
 - [React](react.md)
 - [Vue](vue.md)
+- [Nuxt](vue.md)
```
Comment (caroso1222) [inline recipes/README.md:9]: Typo, I suppose you mean `(nuxt.md)`, right?

File: recipes/nuxt.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# NUXT
+
+## TL;DR
+
+Install and config Notif for [Nuxt](https://nuxtjs.org/)
+
+Exemple in [Codesandbox](https://codesandbox.io/s/nuxt-notyf-s0ugc).
```
Comment (caroso1222) [inline recipes/nuxt.md:7]: Typo, **Example**

File: recipes/nuxt.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# NUXT
+
+## TL;DR
+
+Install and config Notif for [Nuxt](https://nuxtjs.org/)
```
Comment (caroso1222) [inline recipes/nuxt.md:5]: Typo, Notif -> **Notyf**

File: recipes/nuxt.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+Exemple in [Codesandbox](https://codesandbox.io/s/nuxt-notyf-s0ugc).
+
+## Steps
+
+
+1. Add `notif.js` in 📁 plugins
```
Comment (caroso1222) [inline recipes/nuxt.md:12]: Please change `notif.js` for `notyf.js`

File: recipes/nuxt.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+export default {
+...
+css: ["notyf/notyf.min.css"],
+...
+plugins: [{ src: "~/plugins/notyf.js", ssr: false }],
+}
+
+```
+
+4.In your Vue component you can call `this.$notyf`.
```
Comment (caroso1222) [inline recipes/nuxt.md:36]: I think this should be step 5, right?

File: recipes/nuxt.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+```
```
Comment (caroso1222) [inline recipes/nuxt.md:38]: Just wondering, do you not need the language in the block descriptor? 

```
```html
```

**Comment by caroso1222** [comment]
@xlanex6 Thanks a lot for this contribution, looks awesome! there are just a few typos I spotted. Please fix them and we'll be good to go.

File: recipes/nuxt.md (hunk 2 above)
Comment (xlanex6) [reply recipes/nuxt.md:7]: French sorry ;)

**Comment by xlanex6** [comment]
@caroso1222 
All my bad, so sorry about that

Fixed

**Comment by caroso1222** [comment]
Thanks @xlanex6 for the hard work!

**Pull request merged**